### PR TITLE
Update to PF5

### PR DESCRIPTION
**Pull request by ZitaNemeckova**
- [x] dependencies updated to be in compliance with PF5
- [x]  npm ci works
- [x]  npm run start works with warning listed below
- [x] known issue export 'CommonProps' (imported as 'CommonProps') was not found in 'victory-core' error shows in UI
- [ ]  CSS styling issues
- [x]  update README
- [x] remove unused code as much as possible (static?, GH actions?, config?)

### CODE REVIEW

File: src/messages/success.messages.js [hunk 1]
```diff
@@ -45,7 +45,7 @@ const successMessages = defineMessages({
   deployAnsibleText: {
     id: 'success.deployAnsibleText',
     defaultMessage: `<ul>
-<li>First, <a href="https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.2/html/red_hat_ansible_automation_platform_installation_guide/index">install</a> Ansible Automation Platform.</li>
+<li>First, <a>install</a> Ansible Automation Platform.</li>
```
Comment (ZitaNemeckova) [inline src/messages/success.messages.js:48]: Missing a `href`?

File: src/messages/trial.messages.js [hunk 2]
```diff
@@ -165,7 +165,7 @@ const trialMessages = defineMessages({
   },
   startInstallTrial: {
     id: 'trial.startInstallTrial',
-    defaultMessage: 'You may need to wait up to 30 minutes to access <a href="https://cloud.redhat.com/ansible/automation-hub">automation hub</a> services after the trial subscription is activated.'
+    defaultMessage: 'You may need to wait up to 30 minutes to access <a1>automation hub</a1> services after the trial subscription is activated.'
```
Comment (ZitaNemeckova) [inline src/messages/trial.messages.js:168]: `a1` is not a HTML element. Missing a `href`,

File: src/messages/success.messages.js [hunk 3]
```diff
@@ -45,7 +45,7 @@ const successMessages = defineMessages({
...
 <li>Then, <a1>create and launch</a1> your first automation job.</li>
```
Comment (ZitaNemeckova) [inline src/messages/success.messages.js:49]: Probably should be fixed as well.

File: src/components/trial/requirements.js [hunk 4]
```diff
@@ -58,7 +58,7 @@ const Requirements = ({ afterTrial }) => {
                 { intl.formatMessage(trialMessages.reqCardBreakTitle) }
               </Title>
               { intl.formatMessage(startInstallText, {
-                a: (chunks) => <Link link="https://console.redhat.com/ansible/automation-hub">{ chunks }</Link>
+                a1: (chunks) => <Link link="https://console.redhat.com/ansible/automation-hub">{ chunks }</Link>
```
Comment (ZitaNemeckova) [inline src/components/trial/requirements.js:61]: Why `a1`?

File: src/messages/trial.messages.js (hunk 2 above)
Comment (MilanPospisil) [reply src/messages/trial.messages.js:168]: Maybe it is here to distinguish between regular a and element that is being replaced by something else using this function and mapping above. But I would also prefer to name it differently than a or a1.

Comment (himdel) [reply src/messages/trial.messages.js:168]: Don't forget cloud -> console

Comment (ZitaNemeckova) [reply src/messages/trial.messages.js:168]:
```
ARCHITECTURE.md
3:This repo's source code builds a static React app to be served on https://cloud.redhat.com.

src/messages/messages.js
129:        ' collections, and the hosted services on cloud.redhat.com'
146:        'Controller infrastructure is in place, connect it to the hosted services available on cloud.redhat.com.' +
```
and update src/test/components/trial/__snapshots__/trial-pages.test.js.snap snaphot

Comment (himdel) [reply src/messages/trial.messages.js]: Updated both cloud->console and `jest -u` :)

**Comment by himdel** [comment]
Updated packages to latest, except:

* eslint@8 - v9 breaks formatjs
* react-redux@7 - v9 breaks notifications
* redux@4 - v5 breaks notifications
* redux-thunk@2 - v3 needs redux v5
* sass@1.77.6 - 1.77.7+ complains about [mixed-decls](https://sass-lang.com/documentation/breaking-changes/mixed-decls/ ) (which I assume get fixed in pf6)

Also:
removed unused packages, images, scripts, configs
prettier everything, including sorted imports
made eslint pass, disable rules conflicting with prettier, fixed rest
updated react-icons & react-tokens imports to point at individual files
removed index, bootstrap - use AppEntry directly
updated README to match
added a `/preview/` check alongside the `/beta` check (but neither *should* be needed)
made sure no references to cloud.redhat

If there is no dead code or remaining PF issues .. might be ready :)

**Review by MilanPospisil — APPROVED** [review]
LGTM :+1:

**Pull request merged**